appstudio-controller/controllers/appstudio.redhat.com/environment_controller_test.go

```diff
@@ -84,9 +84,10 @@ var _ = Describe("Environment controller tests", func() {
 					Configuration:      appstudioshared.EnvironmentConfiguration{},
 					UnstableConfigurationFields: &appstudioshared.UnstableEnvironmentConfiguration{
 						KubernetesClusterCredentials: appstudioshared.KubernetesClusterCredentials{
-							TargetNamespace:          "my-target-namespace",
-							APIURL:                   "https://my-api-url",
-							ClusterCredentialsSecret: secret.Name,
+							TargetNamespace:            "my-target-namespace",
+							APIURL:                     "https://my-api-url",
+							ClusterCredentialsSecret:   secret.Name,
+							AllowInsecureSkipTLSVerify: true,
```

Comment:
We should probably also add a test that also verifies the case where `AllowInsecureSkipTLSVerify: false`.